### DISCUSSION TITLE
How to add and use Color Picker for picking color from a specific point on screen in WPF?

### DISCUSSION THREAD

**Discussion by mojtabahakimian** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2969] [Q&A]
I have color picker in material design in my project , is color picker have this feature ?  
![image](https://user-images.githubusercontent.com/72439342/203028762-1fc9235d-3cad-45b6-b7a8-2b749bde1e1e.png)

**Comment by Keboo** [comment]
At present, no. There is no feature for picking a color by moving your mouse over another color.
